prqlc/prql-compiler/src/codegen/ast.rs

```diff
@@ -136,10 +136,15 @@ impl WriteSource for ExprKind {
                 Some(r)
             }
             Func(c) => {
-                let mut r = String::new();
+                let mut r = "func ".to_string();
```

Comment:
I guess this is going all-in on having `func`, no strong opposition for the moment